glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -18,6 +19,8 @@ import mozilla.telemetry.glean.net.PingUploader
  *           sent along with all the pings, in the `client_info` section.
  * @property dataPath An optional [String] that specifies where to store data locally on the device.
  *           This should ONLY be used when setting up Glean on a non-main process.
+ * @property logLevel An optional [LevelFilter] that controls how verbose the logging is
+ *           from the rust `log` crate.
```

Comment:
Mentioning "rust log crate" here feels not really informative.
Maybe:

```suggestion
 * @property logLevel An optional [LevelFilter] that controls how verbose the internal logging is.
```